contracts/src/arbitration/KlerosCore.sol

```diff
@@ -482,12 +483,28 @@ contract KlerosCore is IArbitratorV2 {
     /// @param _courtID The ID of the court.
     /// @param _stake The new stake.
     function setStake(uint96 _courtID, uint256 _stake) external {
-        if (!_setStakeForAccount(msg.sender, _courtID, _stake, 0)) revert StakingFailed();
+        if (!_setStakeForAccount(msg.sender, _courtID, _stake, 0, false)) revert StakingFailed();
     }
 
-    function setStakeBySortitionModule(address _account, uint96 _courtID, uint256 _stake, uint256 _penalty) external {
+    function withdrawDelayedStake(uint256 _delayedStakeIndex) external {
```

Comment:
Could we add an event in this function? 🤔  It would make it easier to track on the frontend side.